SCE-Development/SCE-CLI
# Added Docker support and MongoDB container

Add `check_docker_status` in `shared_utils` module
Add MongoDB as a Docker container.
Add `--dbpath` flag to define a path for the data volume used by the MongoDB container. If the target directory does not exist, the script will create the specified directory. Default path is `SCE-CLI/mongo/data/db`. If the desired directory is invalid, the script will abort. The desired directory will be created regardless of Docker's status.

How to test this PR:

- Run `py sce.py setup`
- Run `py sce.py run -s mongo` or `py sce.py run -s mongo --dbpath {custom path}`

The script is able to detect if Docker is installed/running and prompt the user
![image](https://user-images.githubusercontent.com/22359365/153801266-a147444c-81ae-4c77-b12d-116d94054819.png)

The container is running in `-d` (detached) mode. To check the container's status once it is started, use `docker ps`.
